packages/react-instantsearch-hooks-web/src/widgets/SearchBox.tsx

```diff
@@ -40,6 +59,16 @@ export function SearchBox({ queryHook, ...props }: SearchBoxProps) {
     setQuery(event.currentTarget.value);
   }
 
+  function onSubmit(event: React.FormEvent<HTMLFormElement>) {
+    if (!searchAsYouType) {
+      refine(inputValue);
+    }
+
+    if (props.onSubmit) {
+      props.onSubmit(event);
+    }
```

Comment:
```suggestion
    props.onSubmit?.(event);
```
Small suggestion

Reply:
Seems like the ESLint configuration does not allow it, I get a [no-unused-expressions](https://eslint.org/docs/latest/rules/no-unused-expressions) error.
And I think it's best to stay consistent with [the rest here](https://github.com/algolia/react-instantsearch/blob/c7c07f30ac1e7af5559b3fa74855ea001707390f/packages/react-instantsearch-hooks-web/src/ui/SearchBox.tsx#L164-L166)
We neither have `onSubmit?.(event)` nor `inputRef.current?.focus()`

Reply:
No worries, I didn’t tested it, let’s keep it as it is, thanks!